po/fr.po

```diff
@@ -839,18 +840,16 @@ msgid "unknown (bug)"
 msgstr "inconnu (bug)"
 
 #: src/inoreader_urlreader.cpp:34
-#, fuzzy
 msgid "Broadcast items"
-msgstr "Déjà au dernier élément."
+msgstr "Éléments partagés"
 
 #: src/inoreader_urlreader.cpp:35
-#, fuzzy
 msgid "Liked items"
-msgstr "Éléments partagés"
+msgstr "Éléments favoris"
```

Comment:
And replace the translation here with a more litteral one. Again, this may be wrong if users of inoreader "like" items in order to share them...

Reply:
I think you did the right thing. Let's wait for some Inoreader user to tell us if we're wrong here :)